src/utils/ipv6.hpp

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <cstring>
+#include <iterator>
+#include <arpa/inet.h>
+
+#include "ipv6.hpp"
```

Comment:
reference to himself

Reply:
Bad copy-paste from .cpp into .hpp :(

These functions were about the IPV6 connections acceptation but it will be done later (with ipv6.cpp, when we type "bjam targets.jam", we got a unused file...)